java-checks/src/test/files/checks/PseudoRandomCheck.java

```diff
@@ -1,10 +1,52 @@
 import java.util.Random;
+import java.util.concurrent.ThreadLocalRandom;
+import org.apache.commons.lang.math.JVMRandom;
+import org.apache.commons.lang.math.RandomUtils;
+import org.apache.commons.lang.RandomStringUtils;
 
 class A {
   void fun() {
+    int i = nextInt();
+
+    // java.util.Random. Report only constructor
     Random random = new Random(); // Noncompliant [[sc=25;ec=31]] {{Make sure that using this pseudorandom number generator is safe here.}}
     byte[] bytes = new byte[20];
-    random.nextBytes(bytes);
-    double j = Math.random(); // Noncompliant [[sc=21;ec=27]]
+    random.nextBytes(bytes); // Compliant
+
+    // java.lang.Math. Report only Math.random()
+    double rand1 = Math.random(); // Noncompliant [[sc=25;ec=31]]
+    double abs = Math.abs(12); // Compliant
+
+    // java.util.concurrent.ThreadLocalRandom
+    int rand2 = ThreadLocalRandom.current().nextInt();  // Noncompliant
+
+    // org.apache.commons.lang.math.JVMRandom
+    JVMRandom jvmRandom = new JVMRandom(); // Noncompliant
+    double rand3 = jvmRandom.nextDouble(); // Noncompliant
```

Comment:
We should report only 1 issue and not 2.
For class used statically "on the method invocation" (ex: `Math.random()`, `RandomUtils.nextFloat()`)
Otherwise, "on the new class" (ex: `new Random()`, `new JVMRandom()`)